Changelog.md

```diff
@@ -1,5 +1,36 @@
 ## Gazebo 11
 
+## Gazebo 11.11.0 (2022-05-25)
+
+1. Convert `OGRE_RESOURCE_PATH` with `TO_CMAKE_PATH`
+    * [Pull request #3165](https://github.com/ignitionrobotics/gazebo/pull/3165)
```

Comment:
This and the links below need to be updated:

```suggestion
    * [Pull request #3165](https://github.com/osrf/gazebo/pull/3165)
```

Reply:
I do this every time...

Reply:
https://github.com/osrf/gazebo/pull/3221/commits/395b260ec7830d29a4db9d45b6fd8d43bee18d6c